project-koku/koku
# COST-1201: Enable Trino on a per source basis

This changes introduces a new environment variable list `ENABLE_TRINO_SOURCES`.  This contains a list (comma separated) of source uuids that will be processed with the Trino processing pipeline.

**Testing**
1. Start koku-trino with ENABLE_PARQUET_PROCESSING=False.  Ingest an AWS source and verify that Trino is not used.
2.  Add a GCP source and add the source-uuid to the ENABLE_TRINO_SOURCES list.  Restart koku-worker to load variable and hit /download.  Verify source is processed with Trino
3.  Add OCP source and add UUID to trino list.  Verify processing is done with Trino.

**Test Results**
[cost_1201_ut.txt](https://github.com/project-koku/koku/files/6186199/cost_1201_ut.txt)

e2e-deploy: https://github.com/RedHatInsights/e2e-deploy/pull/2899
